### PR TITLE
Change name of the blog to Firefox Addons Review

### LINKED ISSUE #10645: Rename blog to Firefox Add-on Reviews
We link to the new blog on the footer and the More menu. Currently it's just labelled as Blog, but let's call it Firefox Add-on Reviews instead.

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #10645

### CODE REVIEW

File: bin/loading-page.html [hunk 1]
```diff
@@ -60,7 +60,7 @@ <h4 class="Footer-links-header"><a href="/en-GB/firefox/"><!-- react-text: 236 -
                   <!-- /react-text -->
                 </a>
               </li>
-              <li><a href="https://blog.mozilla.com/addons">Blog</a></li>
+              <li><a href="https://blog.mozilla.com/addons">Firefox Add-on Reviews</a></li>
```
Comment (willdurand) [inline bin/loading-page.html:63]: this shouldn't be changed

File: src/amo/components/Footer/index.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ export class FooterBase extends React.Component<InternalProps> {
               </li>
               <li>
                 <a className="Footer-blog-link" href={blogUrl}>
-                  {i18n.gettext('Blog')}
+                  {i18n.gettext('Firefox Add-on Reviews')}
```
Comment (willdurand) [inline src/amo/components/Footer/index.js:74]: It should only be FF Reviews if we link to the new blog.

File: src/amo/components/SectionLinks/index.js [hunk 3]
```diff
@@ -118,7 +118,7 @@ export class SectionLinksBase extends React.Component<InternalProps> {
               prependClientApp={false}
               prependLang={false}
             >
-              {i18n.gettext('Blog')}
+              {i18n.gettext('Firefox Add-on Reviews')}
```
Comment (championshuttler) [inline src/amo/components/SectionLinks/index.js:121]: Same here Will?

**Comment by wagnerand** [comment]
Let's wait until the discussion in the issue has been resolved.

File: src/amo/components/SectionLinks/index.js (hunk 3 above)
Comment (willdurand) [reply src/amo/components/SectionLinks/index.js:121]: This is fine here.

**Review by willdurand — APPROVED** [review]
Looks good, thanks

**Pull request merged**